pkg/util/manifests.go

```diff
@@ -127,7 +128,8 @@ func AppendMissingDefaultMCPManifests(currentMCPs []*mcfgv1.MachineConfigPool) [
 					Name: master,
 				},
 				Spec: mcfgv1.MachineConfigPoolSpec{
-					NodeSelector: v1.AddLabelToSelector(&v1.LabelSelector{}, masterNodeSelector, ""),
+					NodeSelector:          v1.AddLabelToSelector(&v1.LabelSelector{}, masterNodeSelector, ""),
```

Comment:
@eggfoobar please take a look at this part of the change as I think this MCPs lack `MachineConfigSelector`

Reply:
I wonder if we should keep this fix as part of the PR. I know it is needed, but it should link to the proper bug it is fixing.

Reply:
Moved to its own PR #956